CMakeLists.txt

```diff
@@ -75,6 +78,9 @@ if (${ENABLE_THREAD_SANITIZATION})
     add_compile_options(-fsanitize=thread -O1)
     add_link_options(-fsanitize=thread)
     set(CMAKE_EXE_LINKER_FLAGS  "${CMAKE_EXE_LINKER_FLAGS} -fsanitize=thread")
+    add_definitions(-DHYRISE_WITH_TSAN=1)
```

Comment:
Why not use `ENABLE_ADDR_UB_SANITIZATION` or `ENABLE_THREAD_SANITIZATION`?

Reply:
Those are just CMake options, not definitions. Thus, they are not defined and available in the code and we have to add a definition: https://stackoverflow.com/a/12841490

I exported definitions for both and aligned the naming with the `HYRISE_WITH_JEMALLOC` flag.